neurolib/utils/loadData.py

```diff
@@ -15,7 +15,7 @@ class Dataset:
     """Dataset class.
     """
 
-    def __init__(self, datasetName=None, normalizeCmats="max"):
+    def __init__(self, datasetName=None, normalizeCmats="max", fcd=False):
```

Comment:
Can now precompute FCD matrices or not. Added the option because it can take quite some time.